### DISCUSSION TITLE
Availability of CVE-2023-4807 Fix in OpenSSL Versions 3.1.2 and 3.0.8

### DISCUSSION THREAD

**Discussion by mahenderkk08** [openssl/openssl#23850] [Q&A] [answered]
Hi ,

Could someone confirm if the vulnerability mentioned is fixed in OpenSSL versions 3.1.2 and 3.0.8.?

https://nvd.nist.gov/vuln/detail/CVE-2023-4807

#openssl

Thanks & Regards,
Mahender K

**Comment by kulkarniamit** [comment]
Here's some information about this:
* https://www.openssl.org/news/fips-cve.html
* https://www.openssl.org/news/secadv/20230908.txt

**Reply by mahenderkk08** [reply]
No Information , For fix in version 3.1.2 and 3.0.8,  It is mentioned fix in 3.0.11 and 3.1.3 

![image](https://github.com/openssl/openssl/assets/140065778/97c7422a-5a9b-41b7-a56f-7ff4559da054)

**Reply by t8m** [reply, marked as answer]
This vulnerability was fixed in 3.0.11 and 3.1.3 versions so the 3.1.2 and 3.0.8 versions are affected. The FIPS module is not affected by this vulnerability.

**Reply by mahenderkk08** [reply]
Thanks for the confirmation @t8m !

**Comment by mahenderkk08** [comment]
Hi @t8m ,

Could you kindly confirm whether the fix for CVE-2023-4807 has been included in the changes for OpenSSL 3.3.1 and the OpenSSL 3.0.9 FIPS module? I have not been able to find this information.


Thanks!

**Reply by t8m** [reply]
As you can see from the https://openssl-library.org/news/fips-cve/index.html table the FIPS module is not affected by this vulnerability. And as the 3.3.1 release was done much later than the fix releases for CVE-2023-4807, it is also not affected as the code includes the fix.
